### PR TITLE
changed HTML

### PR DESCRIPTION
**Pull request by varonono**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!--The content below is only a placeholder and can be replaced.-->
 <div style="text-align:center">
   <h1>
-    Welcome to {{ title }}!
```
Comment (sziko) [inline src/app/app.component.html]: Delete this line

Comment (sziko) [inline src/app/app.component.html]: Delete this line

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!--The content below is only a placeholder and can be replaced.-->
 <div style="text-align:center">
```
Comment (sziko) [inline src/app/app.component.html]: Change style

Comment (varonono) [reply src/app/app.component.html]: i wont. its pretty

Comment (denisaiernutan) [inline src/app/app.component.html]: sal

File: src/app/app.component.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
   <h1>
-    Welcome to {{ title }}!
+    Welcome to {{ title }}! ON VARO NORBERT BRANCH
   </h1>
   <img width="300" alt="Angular Logo" src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNTAgMjUwIj4KICAgIDxwYXRoIGZpbGw9IiNERDAwMzEiIGQ9Ik0xMjUgMzBMMzEuOSA2My4ybDE0LjIgMTIzLjFMMTI1IDIzMGw3OC45LTQzLjcgMTQuMi0xMjMuMXoiIC8+CiAgICA8cGF0aCBmaWxsPSIjQzMwMDJGIiBkPSJNMTI1IDMwdjIyLjItLjFWMjMwbDc4LjktNDMuNyAxNC4yLTEyMy4xTDEyNSAzMHoiIC8+CiAgICA8cGF0aCAgZmlsbD0iI0ZGRkZGRiIgZD0iTTEyNSA1Mi4xTDY2LjggMTgyLjZoMjEuN2wxMS43LTI5LjJoNDkuNGwxMS43IDI5LjJIMTgzTDEyNSA1Mi4xem0xNyA4My4zaC0zNGwxNy00MC45IDE3IDQwLjl6IiAvPgogIDwvc3ZnPg==">
```
Comment (denisaiernutan) [inline src/app/app.component.html]: hi

**Pull request closed without merging**